_rules/heading-descriptive-b49b2e.md

```diff
@@ -91,17 +84,6 @@ Heading marked up with `role="heading"` that describes the topic or purpose of t
 
 #### Passed Example 3
 
-Heading marked up with `role="heading"` that describes the topic or purpose of the following [palpable content][], with a default aria-level assigned.
-
-```html
-<html lang="en">
-	<span role="heading">Opening Hours</span>
-	<p>We are open Monday through Friday from 10 to 16</p>
-</html>
-```
```

Comment:
@Jym77 This was discussed in the ACT-TF meeting and decided that the example should be removed. There was talk about perhaps making this example inapplicable as I look through the notes here.

It seemed to me that the example was not coherent with the second expectation since the heading was still in the accessibility tree even though it has no accessible name. My thought was an empty accessible name would be sufficient, but I'm open to discussion and curious to hear what others would say.

Reply:
The only difference between this one and passed example 2 is the omission of `aria-level`. The level isn't relevant to this rule, and since omitting `aria-level` is no longer allowed in ARIA 1.2, it didn't seem we should keep the example.